hugo/content/blog/2020-12-24-mumble-1-4-0-snapshot.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Mumble 1.4.0 Snapshot Announcement
+author: Robert Adam
+date: 2020-12-24
+categories:
+  - News
+  - Snapshot
+---
+
+This year we have a Christmas present for you: We are proud to announce that the first **snapshot release of Mumble 1.4.0** is released as of now. It
+is a very important step towards the actual release of version 1.4.0.
+
+After having understood what it means for a release to be a _snapshot release_ (see below), you can download the new version from our [**Downloads
+page**]({{< relref "/downloads" >}}).
+
+<!--more-->
+
+## Snapshot releases
+
+A _snapshot release_ can be considered an open-beta for an upcoming feature release. This means that it contains the latest and greatest features that
```

Comment:
This should be shorter imo. People mostly don't like to read long texts.

Recommendation is:
```
Note: This is an unstable test version (release candidate), so it might have unknown bugs and it maybe does not work correctly. 
We discourage use in productive environments.

Useful Steps before use:
- Backup your configuration files etc. before use

Report any bugs you encounter to our issue tracker.
```

Reply:
Well we never used the term "snapshot release" before so I think a proper explanation of what this is, id due.
And if people don't want to read it, they can still skip it easily. It's not that long after all ^^